exchange/adapter_util.go

```diff
@@ -43,6 +43,13 @@ func buildBidders(infos config.BidderInfos, builders map[openrtb_ext.BidderName]
 			continue
 		}
 
+		if len(info.AliasOf) > 0 {
+			if err := setAliasBuilder(info, builders, bidderName); err != nil {
+				errs = append(errs, fmt.Errorf("%v: failed to set alias builder: %v", bidder, err))
+				continue
+			}
```

Comment:
This code block isn't covered with test coverage, could you update so that it is?

Reply:
@AlexBVolcy We might not be able to test this code yet since we have a check in this function to return an error if `len(info.AliasOf) > 0` since this feature is currently under development. We will add the test case once we are ready to release the feature after testing.